cubed/array_api/inspection.py

```diff
@@ -22,3 +22,11 @@ def devices(self):
 
     def dtypes(self, *, device=None, kind=None):
         return nxp.__array_namespace_info__().dtypes(device=device, kind=kind)
+
+
+def closest_default_dtype(dtype, *, device=None):
```

Comment:
This is not a part of the Array API, so should probably live somewhere else? Also it could do with a unit test.

Reply:
Thanks. I see now how everything in the package is part of the Array API. I'm moving this as a private utility function in the create module for now. If it's needed in more places, I'll find a better place for it later.